src/expr.rs

```diff
@@ -816,10 +814,8 @@ fn rewrite_if_else(context: &RewriteContext,
         _ => " ",
     };
     let mut result = format!("if{}{}{}{}",
-                             between_if_cond_comment.as_ref().map_or(" ", |str| &**str),
-                             pat_expr_string,
-                             after_cond_comment.as_ref().map_or(after_sep, |str| &**str),
-                             if_block_string);
+                             between_if_cond_comment.as_ref().map_or(" ", |str| &**str), pat_expr_string,
+                             after_cond_comment.as_ref().map_or(after_sep, |str| &**str), if_block_string);
```

Comment:
Then I didn't understand how should we format `format!` calls.

We can do like this:
  * If fits single line - do single line
  * If doesn't fit single line - put first argument on first line, and all others on the second
  * If still doesn't fit - do vertical layout

Does it seem correct?